platforms/platform-core/src/index.ts

```diff
@@ -14,14 +14,14 @@ declare module '@jovotech/framework/dist/types/Extensible' {
 
 declare module '@jovotech/framework/dist/types/Jovo' {
   interface Jovo {
-    $corePlatformApp?: CorePlatformApp;
+    $corePlatform?: Core;
   }
 }
-registerPlatformSpecificJovoReference('$corePlatformApp', CorePlatformApp);
+registerPlatformSpecificJovoReference('$corePlatform', Core);
```

Comment:
@rubenaeg
This should be `$core` instead of `$corePlatform` or did we discuss something about this that I've missed?
The same goes for the module augmentation above.

Reply:
You are right, as we discussed I pushed a change that adjusts the code accordingly.